.github/workflows/python-integrate.yaml

```diff
@@ -0,0 +1,172 @@
+# PR's only
+
+name: "Python: Integrate"
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - node/**
+      - go/**
+      - proto/**
```

Comment:
I believe should be just `python` and `proto`

Reply:
Good catch, beginning to think I might need to increase the text size in my IDE